pipeline/ingestor/ingestor_images/src/test/scala/weco/pipeline/ingestor/images/ImagesIngestorFeatureTest.scala

```diff
@@ -88,7 +88,12 @@ class ImagesIngestorFeatureTest
         withImagesIngestor(queue, existingImages = Nil) {
           index =>
             assertElasticsearchEmpty(index)
-            eventually(Timeout(Span(10, Seconds))) {
+            eventually(
```

Comment:
Is the test description right? "does not delete a message from the queue if it fails processing it" then `assertQueueEmpty(queue)` is a bit confusing, unless I'm missing something. Would it be better as "move the message to the DLQ if it fails processing it"?

Reply:
Also I don't quite understand what the interval does that the timeout wasn't already doing 😣

Reply:
The test description is right, but I agree it's misleading.  I started writing a comment to that end, but then I noticed that it was failing due to in-flight messages, so it became less relevant, also, I couldn't think of the right way to say it ("a message fails if it fails" is a bit redundant).

In SQS, a message gets marked invisible until either its visibility timeout expires (it has failed) or it is deleted by the thing that pulled it (it's been successfully processed).  That's what is being tested here - that it does not get deleted (marked successful).

Reply:
The timeout says "keep trying this until it either succeeds or the timeout is reached"
The interval says "wait this long before trying again"

I still don't know why it seemed to be timing out or failing after 5 seconds instead of 10, but it was inspecting the queue over 40 times in those 5 seconds.  It is already known that it will not succeed for at least one second, because that's the value of the visibility timeout on the queue.  That's why I set it to 1s.

Reply:
A reason for not naming this test as you suggest, is that the application does not move it to the DLQ, that's a side effect of it being left on the original queue after the processing attempt.